package.json

```diff
@@ -28,15 +28,22 @@
   "homepage": "https://github.com/pea3nut/short-night#readme",
   "devDependencies": {
     "@types/moveto": "^1.8.0",
-    "node-sass": "^4.11.0",
+    "@types/node": "^17.0.35",
+    "@typescript-eslint/eslint-plugin": "^5.25.0",
+    "@typescript-eslint/parser": "^5.25.0",
+    "eslint": "^8.16.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-airbnb-typescript": "^17.0.0",
+    "eslint-plugin": "^1.0.1",
+    "eslint-plugin-import": "^2.26.0",
+    "sass": "^1.52.1",
     "touch": "^3.1.0",
-    "tslint": "^5.11.0",
-    "tslint-config-airbnb": "^5.11.1",
-    "typedoc": "^0.14.2",
-    "typescript": "^3.0.0"
+    "typedoc": "^0.22.15",
+    "typescript": "^4.6.4"
   },
   "dependencies": {
     "html2canvas": "^1.0.0-rc.5",
     "moveto": "^1.8.0"
-  }
+  },
+  "main": "src"
```

Comment:
I don't think `src/` should be the `main` of Package.json. Can you explain this further?